docs/modules/bing-maps/developer-guide/get-started.md

```diff
@@ -0,0 +1,45 @@
+# deck.gl-bing-maps
+
+This module allows [deck.gl](https://deck.gl) to be used as a Bing Maps custom layer.
+
+## Installation
+
+```bash
+npm install deck.gl deck.gl-bing-maps
```

Comment:
The package.json shows `@deck.gl-community/bing-maps`, is this intentional?

Reply:
The team should vote on that. Reverted for now.